doc/fr/administration_guide/poller/enable_remote.rst

```diff
@@ -35,17 +35,15 @@ mettre à **1** l'option **<no proxy to call Central>**, sinon **0**.
 Cette commande va activer le mode **Remote Server** ::
 
     Starting Centreon Remote enable process:
-
-      Limiting Menu Access...Success
-      Limiting Actions...Done
-
-      Notifying Master...Success
-
-      Set 'remote' instance type...Done
-
-      Centreon Remote enabling finished.
-
-Ajout des droits pour l'utilsateur de base de données centreon d'utiliser la commande **LOAD DATA INFILE**::
-
-    # mysql -h <database_server_address> -u root -p
-    MariaDB [(none)]> GRANT FILE on *.* to 'centreon'@'<remote_server_ip>';
+    Limiting Menu Access...               Success
+    Limiting Actions...                   Done
+    Authorizing Master...                 Done
+    Set 'remote' instance type...         Done
+    Notifying Master...
+    Trying host '10.1.2.3'... Success
+    Centreon Remote enabling finished.
+
+Ajout des droits pour que l'utilisateur de base de données centreon puisse utiliser la commande **LOAD DATA INFILE**::
+
+    # mysql -u root -p
```

Comment:
this won't work on a deported DB like MySQL8
you should restore the previous command

Reply:
Remote Server is not supposed to have a deported database, at least for most installations.